tests/external.py

```diff
@@ -0,0 +1,12 @@
+# simulate an external function which cloudpickle would normally
+# only save a reference to
+mutable_variable = ["return_a_string"]
+mutable_variable2 = ["_nested"]
```

Comment:
The content of those variables does not make the test easy to follow. What about the following (would require the test to be updated accordingly)?

```suggestion
mutable_variable = ["original_value"]
mutable_variable2 = ["original_value2"]
```

Reply:
Have updated names, not using suggestions as I want strings that arent subsets of each other so the in checks can differentiate the two.